packages/apps-config/src/endpoints/productionRelayKusama.ts

```diff
@@ -945,6 +945,8 @@ export const prodParasKusamaCommon: EndpointOption[] = [
     info: 'kusamaCoretime',
     paraId: 1005,
     providers: {
+      // 'IBP-GeoDNS1': 'wss://sys.ibp.network/coretime-kusama',
```

Comment:
Can we add this PR link to the right of each of these as an additional comment like: `// https://github.com/polkadot-js/apps/pull/10514/files`

That way we know why these are commented out.

Reply:
Not needed anymore!! the endpoints are live now 😃 

Many thanks